API.Tests/Parser/ComicParserTests.cs

```diff
@@ -76,6 +77,8 @@ public void ParseComicVolumeTest(string filename, string expected)
         [InlineData("Invincible 033.5 - Marvel Team-Up 14 (2006) (digital) (Minutemen-Slayer)", "33.5")]
         [InlineData("Batman Wayne Family Adventures - Ep. 014 - Moving In", "14")]
         [InlineData("Saga 001 (2012) (Digital) (Empire-Zone)", "1")]
+        [InlineData("spawn-123", "123")]
+        [InlineData("spawn-0", "0")]
```

Comment:
LOL. I meant please add a Volume unit test case for spawn-123 to parse volume as 0.